Controller/Oauth/Index.php

```diff
@@ -131,7 +134,14 @@ public function getMeta()
     public function save(AccountInterface $account)
     {
         $stores = $this->storeRepository->getList();
-        $currentStore = $this->nostoHelperScope->getStore();
+        $storeCode = $this->request->getParam('___store');
```

Comment:
Use constant (https://github.com/Nosto/nosto-magento2/blob/develop/Helper/Url.php#L72) for the store parameter